test/bdd/features/introduce.feature

```diff
@@ -10,284 +10,177 @@ Feature: Introduce protocol
   @skip_proposal
   Scenario: Alice has a Carol's public invitation
     Given   "Bob,Carol" exchange DIDs with "Alice"
-      And   "Alice" sends introduce proposal to the "Bob" with "Carol" invitation
-
+    And   "Alice" sends introduce proposal to the "Bob" with "Carol" invitation
```

Comment:
I believe the end goal of Introduce to have a connection established between 2 parties. Do we have any e2e scenario where one agent(alice) introduce 2 parties(bob and carol) and validate that the DIDExchange/connection between them(bob and carol) is complete ?

Reply:
I have done it. Then I received a comment `It is not a part of the introduce BDD tests`. It is related to did exchange. In these tests, I am checking states id e.g done. It is possible to receive done after did-exchange received an invitation. So, this is how we can be sure that the message was delivered. Was the connection established successfully is not a part of the introduce.

Reply:
May be we don't need to add all the DID-Exchange steps, but we need to validate DIDExchange connection is complete post introduction. These are integration tests and basically we need to show post introduce DIDExchange imo.

@troyronda @llorllale @fqutishat @sandrask thoughts ?

Reply:
@rolsonquadras I think this is a good idea.

Reply:
created an issue to track this - https://github.com/hyperledger/aries-framework-go/issues/1332